aip-console-jenkins/src/main/java/io/jenkins/plugins/aipconsole/AddVersionAction.java

```diff
@@ -20,6 +20,8 @@
     private String filePath;
     private boolean autoCreate = false;
     private boolean cloneVersion = true;
+    private boolean logOutput = true;
```

Comment:
in jenkins i would expect this option to be in the global configuration as the apikey and host, rather than put this in every steps

Reply:
No problem I'll put it there But the global config appeared in all steps
I tested it

Reply:
yes, it should be a global option